src/main/java/org/terasology/funnyblocks/component/BouncyBlockComponent.java

```diff
@@ -0,0 +1,7 @@
+package org.terasology.funnyblocks.component;
+
+import org.terasology.entitySystem.Component;
+
+public class BouncyBlockComponent implements Component {
```

Comment:
Maybe a short javadoc comment on the class here. Something like `/** Responsible for making blocks bouncy, trampoline style! */`